events/community-meeting.md

```diff
@@ -0,0 +1,73 @@
+# Kubernetes Weekly Community Meeting
+
+We have PUBLIC and RECORDED [weekly meeting](https://zoom.us/my/kubernetescommunity) every Thursday at 6pm UTC (1pm EST / 10am PST)
+ 
+Map that to your local time with this [timezone table](https://www.google.com/search?q=1800+in+utc)
+
+See it on the web at [calendar.google.com](https://calendar.google.com/calendar/embed?src=cgnt364vd8s86hr2phapfjc6uk%40group.calendar.google.com&ctz=America/Los_Angeles) , or paste this [iCal url](https://calendar.google.com/calendar/ical/cgnt364vd8s86hr2phapfjc6uk%40group.calendar.google.com/public/basic.ics) into any iCal client.
+
+To be added to the calendar items, join the Google group
+[kubernetes-community-video-chat](https://groups.google.com/forum/#!forum/kubernetes-community-video-chat) for further instructions.
+
+All meetings are archived on the [Youtube Channel](https://www.youtube.com/watch?v=onlFHICYB4Q&list=PL69nYSiGNLP1pkHsbPjzAewvMgGUpkCnJ)
+
+Quick links:
+
+- [Agenda document](https://k8s.devstats.cncf.io/)
```

Comment:
That link is incorrect. This one instead: http://bit.ly/k8scomm18

Reply:
If you do the "k8scomm18" link is the "18" referring to 2018?  If so, that's a maintenance issue to update the link yearly.

The Google Doc itself says "http://bit.ly/k8scommunity "

Reply:
Yeah, the bitlinks are immutable, so this gives the option for an evolving URL.

Reply:
I'm doing separate archive links in order to specifically avoid evolving the URL.